.gitignore

```diff
@@ -49,7 +49,10 @@ coverage.all
 
 /bin
 /dist
-/custom
+/custom/*
+!/custom/conf
+/custom/conf/*
+!/custom/conf/app.ini.sample
```

Comment:
Why add those to `.gitignore`?

Reply:
1. `/custom/conf/app.ini.sample` is tracked in git so should not be in `.gitignore`
2. so I can search `/custom/conf/app.ini.sample` using `ag` or `rg` tools which respect `.gitignore`

Reply:
And no, these 4 lines can not be reduced to 2 because of how `.gitignore` works.

Reply:
@silverwind so those lines are required?

Reply:
Yes, it will not work with just 2 lines, that was my first failed attempt.